Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:latest 
+
+ADD ./package.json package.json
+RUN npm install -g bower
+RUN npm install
+
+ADD ./bower.json bower.json
+RUN bower install --allow-root
+
+RUN openssl req -x509 -new -newkey rsa:2048 -nodes -subj '/C=US/ST=California/L=San Francisco/O=JankyCo/CN=Test Identity Provider' -keyout idp-private-key.pem -out idp-public-cert.pem -days 7300
```

Comment:
I don't think want to generate a new signing credentials as part of dockerfile? This seems like config (e.g. env replacement)